src/main/java/de/tum/in/www1/artemis/domain/AssessmentUpdate.java

```diff
@@ -0,0 +1,36 @@
+package de.tum.in.www1.artemis.domain;
+
+import java.util.List;
+
+/**
+ * A helper object that aggregates a feedback list and a complaint response which is used to update an assessment after
+ * a complaint.
+ */
+public class AssessmentUpdate {
+
+    /**
+     * The updated feedback list
+     */
+    private List<Feedback> feedbacks;
```

Comment:
`feedback` is unaccountable, please call this `feedbackList` (also in the rest of the code, and `getFeedbacks` become `getFeedbackList`)

Reply:
Well I know, but it's called `feedbacks` all over the code, so I tried to be somehow consistent. Of course I can change it, but I don't think that it makes much of a difference. Maybe we should discuss and agree on how to call the feedback list in general.